.github/workflows/mkdocs-main.yml

```diff
@@ -21,10 +21,10 @@ jobs:
 
       - name: install deps
         working-directory: ./python
-        run: cp ../README.md . && pip3 install -r ../requirements-docs.txt && pip3 install -e .[dev]
+        run: cp ../README.md . && pip3 install -e .[dev]
```

Comment:
I think the requirements-docs.txt was trying to fix a CI problem: https://github.com/logicalclocks/hopsworks-api/pull/209

Reply:
Ah, I see, thanks!